src/main/java/org/broadinstitute/hellbender/utils/codecs/gtf/EnsemblGtfCodec.java

```diff
@@ -162,23 +203,27 @@ static boolean validateEnsemblGtfFeature(final GencodeGtfFeature feature) {
      */
     @VisibleForTesting
     boolean validateHeader(final List<String> header, final boolean throwIfInvalid) {
-        if ( header.size() != HEADER_NUM_LINES) {
-            if ( throwIfInvalid ) {
-                throw new UserException.MalformedFile(
-                        "ENSEMBL GTF Header is of unexpected length: " +
-                                header.size() + " != " + HEADER_NUM_LINES);
-            }
-            else {
-                return false;
+        // As it turns out, the ENSEMBL GTF header is pretty loosy-goosy.
+        // No fields are required, and therefore it could actually be empty.
+
+        // Rather than attempting to validate the file, here we just
+        // assert that all header lines begin with a comment (they should already).
```

Comment:
Are there fields that YOU require even if ENSEMBLE doesn't?  It might be a good idea to warn if things like the version are missing.

Reply:
I would love to make it more restrictive, but I don't want to require users to modify the files after downloading them.  If we end up writing a downloader tool that could create these metadata entries.

I'm not sure adding in warnings will be productive - users should never be creating these files, they should just be downloading them directly from ENSEMBL.  So it's really an ENSEMBL issue.

Reply:
That seems reasonable.